# patches the version of the software which currently is an issue
# see https://github.com/CH-Earth/summa/issues/485
--- build/Makefile.original	2021-08-19 12:56:30.191385895 -0500
+++ build/Makefile	2021-08-19 12:56:56.079512711 -0500
@@ -300,7 +300,7 @@
 
 # Define version number
 VERSIONFILE = $(DRIVER_DIR)/summaversion.inc
-VERSION = $(shell git tag | tail -n 1)
+VERSION = "v3.0.3"
 BULTTIM = $(shell date)
 GITBRCH = $(shell git describe --long --all --always | sed -e's/heads\///')
 GITHASH = $(shell git rev-parse HEAD)
